### PR TITLE
SUBS-709 Adds ability for dynamic carousel in dynamic contentful hero

### PR DESCRIPTION
**Pull request by eddieferrer**

### CODE REVIEW

**Comment by eddieferrer** [comment]
https://user-images.githubusercontent.com/4371888/117069430-4f1f4300-ace1-11eb-979b-1a915b28fb8f.mov

**Comment by eddieferrer** [comment]
I wasn't sure which style of carousel, we wanted here, so went with mostly default options as seen in video

**Review by mcstover — APPROVED** [review]
Looks good!

Did this require any contentful updates or were you able to handle that in the code?

**Comment by eddieferrer** [comment]
> Looks good!
> 
> Did this require any contentful updates or were you able to handle that in the code?

@mcstover no contentful updates needed

File: src/components/Contentful/DynamicHero.vue [hunk 1]
```diff
@@ -79,28 +97,49 @@
 <script>
 import KvButton from '@/components/Kv/KvButton';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import KvCarousel from '@/components/Kv/KvCarousel';
```
Comment (ryan-ludwig) [inline src/components/Contentful/DynamicHero.vue:100]: Could these be dynamically imported depending on `isHeroCarousel` ?

Comment (eddieferrer) [reply src/components/Contentful/DynamicHero.vue:100]: Yeah possibly? I can  do that in a followup PR. Are you concerned about loading embla and carousel when it's not needed?

Comment (ryan-ludwig) [reply src/components/Contentful/DynamicHero.vue:100]: Yeah, exactly. Embla has some weight. More broadly working towards dynamically importing only the components in use when we have conditional logic like experiments and Contentful driven stuff.

Comment (ryan-ludwig) [reply src/components/Contentful/DynamicHero.vue:100]: I also realize that Embla is currently loaded everywhere in our vendors bundle right now, but we should be able to sort that out separately.

**Pull request merged**